presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveStorageFormats.java

```diff
@@ -465,6 +546,21 @@ private void setAdminRole(Connection connection)
         }
     }
 
+    /**
+     * Ensures that a view named "dummy" with exactly one row exists in the default schema.
+     */
+    // These tests run on versions of Hive (1.0.0 on CDH 5) that don't fully support SELECT without FROM
+    // (particularly when using certain forms of authentication).
+    private void ensureDummyExists()
+            throws SQLException
+    {
+        onHive().executeQuery("CREATE TABLE IF NOT EXISTS `_dummy_table_` (`_dummy_column_` varchar(1))");
+        onHive().executeQuery("INSERT INTO `_dummy_table_` VALUES ('x')");
+        onHive().executeQuery(
+                "CREATE VIEW IF NOT EXISTS dummy (`_dummy_column_`)"
+                        + "AS SELECT 'x' FROM `_dummy_table_` LIMIT 1");
```

Comment:
Let's not implicate views here. Let's just have single-row table.

I see you're concerned someone will add a second row to the table and thus it won't be single-row table anymore, but that would be rogue and tests would fail telling us about that. thus i wouldn't be concerned about that.

Reply:
Already done. I just re-create the table each time.